<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(87, 227, 137);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="shockButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 97, 81);</string>
    </property>
    <property name="text">
     <string>SHOCK</string>
    </property>
   </widget>
   <widget class="QWidget" name="ecgDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>501</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="shockCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SHOCKS: 05</string>
     </property>
    </widget>
    <widget class="QLabel" name="elapsedTime">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>05:25</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>219</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CHECK RESPONSIVENESS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>501</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="heartDisplay"/>
    </widget>
   </widget>
   <widget class="QWidget" name="cprDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="cprTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-1</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CPR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="cprSimLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="statusDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>STATUS INDICATOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="statusPassed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="interactive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGraphicsView" name="statusFailed">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 193, 241);</string>
     </property>
     <property name="text">
      <string>on/off</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
